qmf/src/libraries/qmfclient/qmailmessage.cpp

```diff
@@ -1153,6 +1156,20 @@ namespace findAttachments
             if (container.multipartType() == QMailMessagePart::MultipartMixed)
                 inMultipartMixed(container, found, hasAttachments);
 
+            if (container.multipartType() == QMailMessagePart::MultipartSigned) {
```

Comment:
Put this loop into a 'inMultipartSigned' function to be consistent with the existing code.